### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by Sven792**

### CODE REVIEW

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Reiseführer
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: I don't think this properly represents the meaning of the name.

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+item.patchouli:test_book_2.name=Testbuch 2
+item.patchouli:test_book_2.landing=Einführungstext! Es unterstützt auch $(2)Farben$() und $(4)solche $(bold)Sachen$()!
+
+item.patchouli:test_completion.name=Fortschritts-Testbuch
+item.patchouli:test_completion.landing=Das Buch ändert die Farben, wenn du Einträge freischaltest!
+
+item.patchouli:intro_book.name=Patchouli
+item.patchouli:intro_book.subtitle=Dokumentation für jedermann
+item.patchouli:intro_book.landing=Mit $(item)Patchouli$() machst du es dir einfach, fortschrittliche freischaltbare $(thing)Bücher$(), zu lesen für Modifikationen und Modifikationenpakete!
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: The second comma is misplaced. Also, I really don't like how you translated "mod" and "mod pack", I don't think those should be translated at all.

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+# Testing
+item.patchouli:test_book_1.name=Testbuch 1
+item.patchouli:test_book_1.landing=Nicht so toll wie eine Einführung, es ist aber trotzdem präsent.
+
+item.patchouli:test_book_2.name=Testbuch 2
+item.patchouli:test_book_2.landing=Einführungstext! Es unterstützt auch $(2)Farben$() und $(4)solche $(bold)Sachen$()!
+
+item.patchouli:test_completion.name=Fortschritts-Testbuch
+item.patchouli:test_completion.landing=Das Buch ändert die Farben, wenn du Einträge freischaltest!
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
item.patchouli:test_completion.landing=Das Buch ändert seine Farben, wenn du Einträge freischaltest!
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 4]
```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Reiseführer
+
+# Testing
+item.patchouli:test_book_1.name=Testbuch 1
+item.patchouli:test_book_1.landing=Nicht so toll wie eine Einführung, es ist aber trotzdem präsent.
+
+item.patchouli:test_book_2.name=Testbuch 2
+item.patchouli:test_book_2.landing=Einführungstext! Es unterstützt auch $(2)Farben$() und $(4)solche $(bold)Sachen$()!
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
item.patchouli:test_book_2.landing=Einführungstext! Es unterstützt auch $(2)Farben$() und $(4)andere $(bold)Sachen$()!
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 5]
```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Reiseführer
+
+# Testing
+item.patchouli:test_book_1.name=Testbuch 1
+item.patchouli:test_book_1.landing=Nicht so toll wie eine Einführung, es ist aber trotzdem präsent.
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: This is really bad German.
```suggestion
item.patchouli:test_book_1.landing=Nicht so toll wie eine Einführung, ist aber trotzdem präsent.
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+item.patchouli:test_completion.name=Fortschritts-Testbuch
+item.patchouli:test_completion.landing=Das Buch ändert die Farben, wenn du Einträge freischaltest!
+
+item.patchouli:intro_book.name=Patchouli
+item.patchouli:intro_book.subtitle=Dokumentation für jedermann
+item.patchouli:intro_book.landing=Mit $(item)Patchouli$() machst du es dir einfach, fortschrittliche freischaltbare $(thing)Bücher$(), zu lesen für Modifikationen und Modifikationenpakete!
+
+
+#Subtitles
+patchouli.subtitle.book_open=Buch, das geöffnet wird
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.subtitle.book_open=Buch wird geöffnet
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.subtitle.book_flip=Seitenumschläge
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: This sounds like you've used google translate, even though I know you haven't... These are subtitles, so please translate them as such.
```suggestion
patchouli.subtitle.book_flip=Umblättern
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+
+
+# Gui Strings
+patchouli.gui.lexicon.landing_info=(Platzhalter für den Einführungstext, bestimme dein eigenes in der json-Datei deines Buches!)
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: You are aware that genus exists, yes?
```suggestion
patchouli.gui.lexicon.landing_info=(Platzhalter für den Einführungstext, bestimme deinen eigenen in der json-Datei deines Buches!)
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.progress_meter=Fortschritt freischalten:
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: "unlock" is not a verb here... Not sure it's even necessary to have translated at all, so here are two suggestions.
```suggestion
patchouli.gui.lexicon.progress_meter=Fortschritt:
```
```suggestion
patchouli.gui.lexicon.progress_meter=Freischaltungs-Fortschritt:
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.progress_tooltip=%d/%d Freigeschaltet
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.progress_tooltip=%d/%d freigeschaltet
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.progress_tooltip.secret1=Plus 1 Geheimnis
+patchouli.gui.lexicon.progress_tooltip.secret=Plus %d Geheimnisse
+patchouli.gui.lexicon.progress_tooltip.info=Mache Fortschritte um noch mehr freizuschalten!
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: This is how Vanilla translates it. Please stay close to the Vanilla translation where appropriate.
```suggestion
patchouli.gui.lexicon.progress_tooltip.info=Erziele Fortschritte, um noch mehr freizuschalten!
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.categories=Kategorien
+patchouli.gui.lexicon.chapters=Kapitel
+patchouli.gui.lexicon.editor=Buch-Bearbeitungsprogramm
+patchouli.gui.lexicon.editor.mock_header=Kopfzeile der Nachbildung
+patchouli.gui.lexicon.editor.info=Dies ist ein Bearbeitungsprogramm für Patchouli's-Bucheinträge. Damit ist gemeint, dass es für die Entwicklung oder Übersetzung genutzt wird. Es gibt dafür kein Nutzen wenn du ein Spieler bist.$(br2)Bitte achte auf dem $(l:https://github.com/Vazkii/Patchouli/wiki/Text-Formatting-101)Patchouli Wiki$() für nützliche Steuercodes.
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.editor.info=Dies ist ein Bearbeitungsprogramm für Patchouli-Bucheinträge. Es ist für die Entwicklung oder Übersetzung gedacht. Wenn du ein Spieler bist, ist es nutzlos.$(br2)Besuche das $(l:https://github.com/Vazkii/Patchouli/wiki/Text-Formatting-101)Patchouli Wiki$() für nützliche Steuercodes.
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.shift_for_recipe=Umschalttaste-klicken für Rezept
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: I don't like this, but I cannot think of a much better translation...

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.index=Eintragsverzeichnis
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: Maybe simplify it? Not sure whether this is better or worse.
```suggestion
patchouli.gui.lexicon.index=Verzeichnis
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.index.info=Ein Verzeichnis von jedem Kapitel ist Verfügbar in diesem Buch.$(br2)Du kannst den Verzeichnis durchsuchen oder irgendwelche andere Kategorie, ganz einfach durch das Eintippen der Anfrage. Die Suchleiste wird dann erscheinen um dich zu unterstützen.
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: This translation is just completely wrong. Please read the original correctly when translating
```suggestion
patchouli.gui.lexicon.index.info=Ein Verzeichnis aller in diesem Buch zu findenden Kapitel.$(br2)Du kannst das Verzeichnis sowie jegliche andere Kategorie durchsuchen, indem zu anfängst, Begriffe einzutippen. Eine Suchleiste wird dann erscheinen, um dir bei der Suche zu helfen.
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.shapeless=Formlos
+patchouli.gui.lexicon.no_results=Keine Ergebnisse
+patchouli.gui.lexicon.sad=:(
+patchouli.gui.lexicon.locked=(Gesperrt)
+patchouli.gui.lexicon.reloaded=Buch neugeladen. Dauerte %d ms.
+patchouli.gui.lexicon.add_bookmark=Lesezeichen hinzufügen
+patchouli.gui.lexicon.remove_bookmark=(Umschalttaste drücken und klicken zum entfernen)
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.remove_bookmark=(Umschalttaste drücken und klicken zum Entfernen)
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.multiblock_bookmark=Visualisierte Struktur
+patchouli.gui.lexicon.relations=Verwandte Kapitel
+patchouli.gui.lexicon.not_anchored=Rechts-klicken auf einen Block um die Struktur zu fixieren
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.not_anchored=Rechts-klicken auf einen Block, um die Struktur zu fixieren
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.structure_complete=Fertig!
+patchouli.gui.lexicon.needs_air=(Lösche Blöcke die rot markiert sind)
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.needs_air=(Lösche Blöcke, die rot markiert sind)
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.visualize_letter=V
+patchouli.gui.lexicon.seconds=%ss
+patchouli.gui.lexicon.history=Lese-Verlauf
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.history=Leseverlauf
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.history.info=Dieser Bereich speichert einige der vorherigen Kapiteln die du besucht hast.$(br2)Kapiteln werden automatisch hinzugefügt und entfernt während du das Buch durchblätterst. Willst du aber, dass sie länger referenziert bleiben, kannst du sie als Lesezeichen hinzufügen. $(br2)$(o)Nützlicher Hinweis: Versuche mal die Umschalttaste zu drücken und klicke auf der Kapitel-Schaltfläche!$()
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: The plural of "Kapitel" is "Kapitel", not "Kapiteln"... And you're missing punctuation again.
```suggestion
patchouli.gui.lexicon.history.info=Dieser Bereich speichert die vorherigen paar Kapitel, die du besucht hast.$(br2)Kapitel werden automatisch hinzugefügt und entfernt, während du das Buch durchblätterst. Willst du aber, dass sie länger gelistet bleiben, kannst du sie als Lesezeichen hinzufügen. $(br2)$(o)Nützlicher Hinweis: Versuche mal, die Umschalttaste zu drücken, und dann dann auf die Kapitel-Schaltfläche zu klicken!$()
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 21]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.toast=Neue Kapitel
+patchouli.gui.lexicon.toast.info=Neue Kapitel Freigeschaltet
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.toast.info=Neue Kapitel freigeschaltet
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 22]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.external_link=(Externer Link)
+patchouli.gui.lexicon.loading_error=Lade Fehler!
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: ...what?
```suggestion
patchouli.gui.lexicon.loading_error=Ladefehler!
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 23]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.loading_error_hover=(Schwebe für Info)
+patchouli.gui.lexicon.loading_error_log=Prüfe deinen Protokoll für mehr
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.loading_error_log=Sieh im Protokoll-Log nach
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 24]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.dev_edition=Schreiber
+patchouli.gui.lexicon.edition_str=%s Edition
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: It looks to me like you didn't even check this...
```suggestion
patchouli.gui.lexicon.edition_str=%s-Edition
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 25]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.added_by=Hinzugefügt von %s
+patchouli.gui.lexicon.sneak=Schleiche zum anzeigen
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.sneak=Schleiche zum Anzeigen
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 26]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.view=Zeige Eintrag
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.view=Eintrag anzeigen
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 27]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.keybind=Tastenbelegung: %s
+patchouli.gui.lexicon.keybind_missing=Keine solche Tastenbelegung: %s
+patchouli.gui.lexicon.objective=Zielsetzung
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.objective=Ziel
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 28]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.incomplete=Unvollständig
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.incomplete=Nicht erreicht
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 29]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.complete=Vollständig!
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.complete=Erreicht!
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 30]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.mark_complete=Markierung Vollständig
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: Why would you translate "mark" as a noun here?
```suggestion
patchouli.gui.lexicon.mark_complete=Als erreicht markieren
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 31]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.mark_incomplete=Markierung Unvollständig
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.mark_incomplete=Als nicht erreicht markieren
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 32]
```diff
@@ -0,0 +1,93 @@
...
+
+patchouli.gui.lexicon.button.prev_page=Vorher
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: Rather unfortunate that we already have a "Zurück" button.
```suggestion
patchouli.gui.lexicon.button.prev_page=Vorherig
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.next_page=Nächste
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.button.next_page=Weiter
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 34]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.back=Zurück
+patchouli.gui.lexicon.button.back.info=Umschalttaste drücken und klicken um nach Hause zurückzukehren
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: ... now this _really_ sounds like google translate.
```suggestion
patchouli.gui.lexicon.button.back.info=Umschalttaste drücken und klicken, um zum Hauptmenü zurückzukehren
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 35]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.resize=Größe ändern Benutzerschnittstelle
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.button.resize=Größe ändern
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 36]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.resize.size0=Standard
+patchouli.gui.lexicon.button.resize.size1=Klein
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.button.resize.size1=Winzig
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 37]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.resize.size2=Mittel
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.button.resize.size2=Klein
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 38]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.resize.size3=Mittelgroß
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.button.resize.size3=Mittel
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 39]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.resize.size4=Groß
+patchouli.gui.lexicon.button.resize.size5=Rießig
+patchouli.gui.lexicon.button.resize.size6=Wie groß ist denn dein Bildschirm?
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: This is a joke, you know.
```suggestion
patchouli.gui.lexicon.button.resize.size6=Wie riesig ist dein Bildschirm?
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 40]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.editor=Bearbeitungsprogramm
+patchouli.gui.lexicon.button.editor.info=(Umschalttaste drücken und klicken zum neuladen)
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.button.editor.info=(Umschalttaste drücken und klicken zum Neu-laden)
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 41]
```diff
@@ -0,0 +1,93 @@
...
+patchouli.gui.lexicon.button.advancements=Fortschritte
+patchouli.gui.lexicon.button.config=Konfiguration
+patchouli.gui.lexicon.button.visualize=Visualisieren
+patchouli.gui.lexicon.button.visualize.info=(Klicke nochmal zum zurücksetzen)
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]:
```suggestion
patchouli.gui.lexicon.button.visualize.info=(Klicke nochmal zum Zurücksetzen)
```

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 11 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: I know. I missed it.

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 14 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: I don't know.

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 22 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: Thinking errors... :D

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 23 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: Ok... If I think about my translation now, it looks strange... But "log" is "Protokoll"?

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 9 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: Hmm... the first suggestion is better I mean.

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 30 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: Hmm... This is a good question.

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 34 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: Oh... "Home" with THAT sense....

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 2 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: However, the word "mod" is translated correctly. "Mod" is an unofficially abbreviation in german.

File: src/main/resources/assets/patchouli/lang/de_DE.lang (hunk 1 above)
Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: Hmm... "Anleitungsbuch" would be better I think.

**Comment by Sven792** [comment]
@Vexatos ?

File: src/main/resources/assets/patchouli/lang/de_DE.lang [hunk 42]
```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Anleitungsbuch
```
Comment (Vexatos) [inline src/main/resources/assets/patchouli/lang/de_DE.lang]: Well this book might contain anything, that's the issue here; this sounds too much like an instruction manual. I am still not happy with the name...

Comment (Sven792) [reply src/main/resources/assets/patchouli/lang/de_DE.lang]: Is "Handbuch" an option? @Vexatos

**Comment by Sven792** [comment]
What did you mean? @Vexatos

**Pull request merged**